data.sql

```diff
@@ -35,3 +35,30 @@ UPDATE animals SET owner_id = 3 WHERE name IN ('Devimon', 'Plantmon');
 UPDATE animals SET owner_id = 4 WHERE name IN ('Charmander', 'Squirtle', 'Blossom'); 
 UPDATE animals SET owner_id = 5 WHERE name IN ('Angemon', 'Boarmon'); 
 
+-- Insert the following data for vets -- 
+
+INSERT INTO vets(name,age,date_of_graduation)
+VALUES('William Tatcher', 45, '2000-04-23') 
+,('Vet Maisy Smith', '26',  date '2019-01-17'),
+('Vet Stephanie Mendez', '64',  date '1981-05-04'),
+('Vet Jack Harkness', '38',  date '2008-06-08');
+
+-- Insert the data for specialties: 
+INSERT INTO specializations(vets_id,species_id) VALUES ('1', '1'),('3', '1'),('3', '2'),('4', '2');
+
+-- Insert the following data for visits:
+
+INSERT INTO visits(animals_id, vets_id, date_of_visit)
+VALUES
+	('1', '1', date '2020-05-24'),
+	('1', '3', date '2020-07-22'),
+	('2', '4', date '2021-02-02'),
+	('3', '2', date '2020-01-05'),
+	('3', '2', date '2020-03-08'),
+	('3', '2', date '2020-05-14'),
+	('4', '3', date '2021-05-04'),
+	('5', '4', date '2021-02-24'),
+	('6', '2', date '2019-12-21'),
+	('6', '1', date '2020-08-10'),
+	('6', '2', date '2021-04-07'),
+	('7', '3', date '2019-09-29');
```

Comment:
- [ ]Great job but its like you missed to insert part of the data
<img width="1059" alt="image" src="https://user-images.githubusercontent.com/37901469/197208831-0b617551-7276-4725-a993-64509c195c57.png">